README.md

```diff
@@ -1,8 +1,9 @@
-# momo-scheduler <img src="momo_logo.svg" align="right" />
 
-momo is a light-weight, easy-to-use interval-based scheduler that persists jobs in a MongoDB.
+# momo-scheduler <img src="momo_logo.svg" align="right" />
 
-In essence, it provides an easy way to tell your application to "run that job every five minutes".
+momo is a light-weight, easy-to-use scheduler that persists jobs in a MongoDB and supports interval-based scheduling as
+well as cron syntax. In essence, it provides an easy way to tell your application to either "run that job every five
+minutes" or "run that job according to this cron schedule".
```

Comment:
maybe also give a concrete example for the cron (as for the interval)?
Like 
In essence, it provides an easy way to tell your application to "run that job every five
minutes" or "run that job every weekday at 8 AM"

Reply:
Good idea!